project/Build.scala

```diff
@@ -2057,14 +2057,14 @@ object Build {
           case `default212Version` =>
             if (!useMinifySizes) {
               Some(ExpectedSizes(
-                  fastLink = 625000 to 626000,
+                  fastLink = 626000 to 627000,
```

Comment:
Why does this increase the code size? I would not have expected this.

Reply:
Benign local variable name changes that tipped the size from just under to just above the threshold. After renaming by GCC or any other minifier, the difference disappears.